code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -439,3 +439,42 @@
 		M.emote(pick("twitch","laugh","frown"))
 	..()
 	. = 1
+
+/datum/reagent/drug/acid
+	name = "Acid"
+	description = "A strong psychedelic drug."
+	reagent_state = LIQUID
+	color = "#B0B0B0"
+	taste_description = "bitterness"
+
+/datum/reagent/drug/acid/on_mob_life(mob/living/M)
```

Comment:
Technically you could use a mixture of spawn while sleep to get it to update more often than every 2 seconds

Reply:
that'd be hacky